### ISSUE TITLE
auto-generated message ID override id in V3 

### ISSUE THREAD

**Issue by meysamizadmehr** [lingui/js-lingui#604]
**Describe the bug**
Auto-generated message ID overrides `id` property in the `Trans` component.  This component
` <Trans id="msg.hello">Hello World</Trans>` will transfformed to ` <Trans id="Hello World"></Trans>`.

**To Reproduce**

```
import { Trans } from "@lingui/react"

export default function App() {
   return <Trans id={"msg.hello"}>Hello World</Trans>
}
```

**Expected behavior**
The `id` should override the auto-generated message ID.

**Additional context**

- jsLingui version ``3.0.0-7``

.linguirc
```
{
  "catalogs": [
    {
      "path": "<rootDir>/translations/{locale}/messages",
      "include": [
        "<rootDir>"
      ],
      "exclude": [
        "*/node_modules/*"
      ]
    }
  ],
  "locales": [
    "de",
    "es"
  ],
  "format": "po"
}
```

**Comment by arcanis** [comment]
Together with `t` not supporting ids anymore, it sounds like custom ids aren't possible at all with the new macro implementations.

**Comment by tricoder42** [comment]
Custom ids in `Trans` should be supported. `t` macro doesn't support them - you should use `defineMessage` instead, which is more verbose.

**Comment by tricoder42** [comment]
What happens if you remove expression parenthesis? i.e. `<Trans id="msg.hello">Hello World</Trans>`

**Comment by tricoder42** [comment]
Fixed in #682
